Tests/SwiftParserTest/Assertions.swift

```diff
@@ -555,7 +556,18 @@ func AssertParse<S: SyntaxProtocol>(
   var (markerLocations, source) = extractMarkers(markedSource)
   markerLocations["START"] = 0
 
-  let tree: S = parse(source)
+  let enableTestCaseMutation = false
+  // FIXME: Currently, tests are failing when we enable test case mutation.
+  // Once all of those issues are fixed, this should become
+  // let enableTestCaseMutation = ProcessInfo.processInfo.environment["SKIP_LONG_TESTS"] != "1"
```

Comment:
Out of interest, how much longer does this take to run?

Reply:
Oh, it’s quite significant. It increases test time from 39s to 250s if you are not skipping long tests (`SKIP_LONG_TESTS=1`)

Reply:
How many mutations happen? Given it's mutating one at a time (concurrently, but each one just has the one change), That's maybe not as bad as I thought it would be 😅. Wonder if it's worth batching some together.

Reply:
> How many mutations happen?

I didn’t measure but I imagine there are on average 3 to 5 different choices for each token. I thought about batching them together, but you really need to be clever so one doesn’t shadow the other. And trying to be clever is usually a bad idea in my experience.